### PR TITLE
Add POST and PUT examples in roles API doc

### PR DESCRIPTION
**Pull request by hillaryfraley**
Add POST /roles and PUT /roles/:role examples.

Epic #1452

### CODE REVIEW

File: content/sensu-go/5.16/api/roles.md [hunk 1]
```diff
@@ -27,10 +27,29 @@ The `/roles` API endpoint provides HTTP GET access to [role][1] data.
 The following example demonstrates a request to the `/roles` API endpoint, resulting in a JSON array that contains [role definitions][1].
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/roles -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET http://127.0.0.1:8080/api/core/v2/namespaces/default/roles
```
Comment (Blue0ctober) [inline content/sensu-go/5.16/api/roles.md]: requires ' \ ' at the end of the line, probably should move 'curl -X GET' to it's own line for consistency

File: content/sensu-go/5.16/api/roles.md [hunk 2]
```diff
@@ -125,7 +194,8 @@ The `/roles/:role` API endpoint provides HTTP GET access to [role data][1] for s
 In the following example, querying the `/roles/:role` API endpoint returns a JSON map that contains the requested [`:role` definition][1] (in this example, for the `:role` named `read-only`).
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/roles/read-only -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET http://127.0.0.1:8080/api/core/v2/namespaces/default/roles/read-only
```
Comment (Blue0ctober) [inline content/sensu-go/5.16/api/roles.md]: requires ' \ ' at the end of the line, probably should move 'curl -X GET' to it's own line for consistency

**Review by Blue0ctober — CHANGES_REQUESTED** [review]
See comments for recommended changes

File: content/sensu-go/5.16/api/roles.md (hunk 1 above)
Comment (hillaryfraley) [inline content/sensu-go/5.16/api/roles.md]:
```suggestion
curl -X GET \
http://127.0.0.1:8080/api/core/v2/namespaces/default/roles \
```

File: content/sensu-go/5.16/api/roles.md (hunk 2 above)
Comment (hillaryfraley) [inline content/sensu-go/5.16/api/roles.md]:
```suggestion
curl -X GET \
http://127.0.0.1:8080/api/core/v2/namespaces/default/roles/read-only \
```

**Pull request merged**